automatic/vscode-insiders.install/tools/ChocolateyInstall.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+$softwareName = 'Microsoft Visual Studio Code Insiders'
+$version = '1.46.0'
+if ($version -eq (Get-UninstallRegistryKey "$softwareName").DisplayVersion) {
+  Write-Host "VS Code Insiders $version is already installed."
+  return
+}
+
+$pp = Get-PackageParameters
+Close-VSCodeInsiders
+
+$packageArgs = @{
+  packageName    = 'vscode-insiders.install'
```

Comment:
you can use `$env:chocolateyPackageName` instead of explicitly setting the name itself. (`vscode` used it this way because the environment variable didn't exist when it was created).